.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: Main workflow
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          # - windows-latest
+        ocaml-version:
+          - 4.12.x
+          - 4.11.x
+          - 4.10.x
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Use OCaml ${{ matrix.ocaml-version }}
+        uses: actions-ml/setup-ocaml@master
+        with:
+          ocaml-version: ${{ matrix.ocaml-version }}
+          opam-depext: false
+          opam-pin: false
+
+      - run: |
+          opam pin add owl-ode-base.dev . --no-action
+          opam pin add owl-ode.dev . --no-action
+          opam pin add owl-ode-odepack.dev . --no-action
+
+      - run: opam depext --yes --with-test owl-ode-base owl-ode owl-ode-odepack
```

```suggestion
      - run: |
          brew list
          opam depext --yes --with-test owl-ode-base owl-ode owl-ode-odepack
          brew list
          ls /usr/local/bin
          gfortran --version
          which gfortran
```
could you add this temporarily for testing purpose?

---

Sure thing!